cmd/cdi-controller/controller.go

```diff
@@ -75,6 +66,14 @@ func init() {
 	glog.V(Vdebug).Infof("init: complete: cdi controller will create importer using image %q\n", importerImage)
 }
 
+func getRequiredEnvVar(name string) string {
+	val := os.Getenv(name)
```

Comment:
You could just use:  `v, ok := os.LookupEnv(name)` here, I realize it doesn't matter because in this case we're treating not-set and empty the same, but it's more explicit at any rate, not a big deal though.

Reply:
Fair enough